### PR TITLE
Provide HtmlEditor table operation shortcuts for the toolbar module

### PR DESCRIPTION
**Pull request by DokaRus**
image
After focus the table:
image
Insert Table dialog:
image

### CODE REVIEW

File: js/ui/html_editor/modules/toolbar.js [hunk 1]
```diff
@@ -279,6 +303,76 @@ if(Quill) {
             };
         }
 
+        get _insertTableFormItems() {
```
Comment (ksercs) [inline js/ui/html_editor/modules/toolbar.js:306]: did u mean `_insertedTableFormItems`?

File: testing/tests/DevExpress.ui.widgets.editors/htmlEditorParts/api.tests.js [hunk 2]
```diff
@@ -25,7 +26,32 @@ const moduleConfig = {
...
+        };
+
+        this.createEditor = () => {
+            this.instance = $('#htmlEditor')
+                .dxHtmlEditor(this.options)
+                .dxHtmlEditor('instance');
+        };
+
+        this.applyTableMethod = (methodName, ...restArgs) => {
+            return this.instance && this.instance.getModule('table')[methodName](...restArgs);
```
Comment (ksercs) [inline testing/tests/DevExpress.ui.widgets.editors/htmlEditorParts/api.tests.js:44]:
```suggestion
            return this.instance?.getModule('table')[methodName](...restArgs);
```

File: testing/tests/DevExpress.ui.widgets.editors/htmlEditorParts/toolbarModule.tests.js [hunk 3]
```diff
@@ -1089,6 +1123,41 @@ testModule('Toolbar dialogs', dialogModuleConfig, () => {
...
+        new Toolbar(this.quillMock, this.options);
+        this.$element
+            .find(`.${TOOLBAR_FORMAT_WIDGET_CLASS}`)
+            .trigger('dxclick');
+
+        const $form = $(`.${FORM_CLASS}`);
+        const $fields = $form.find(`.${FIELD_ITEM_CLASS}`);
+        const fieldsText = $form.find(`.${FIELD_ITEM_LABEL_CLASS}`).text();
+
+        assert.equal($fields.length, 2, 'Form with 2 fields shown');
```
Comment (ksercs) [inline testing/tests/DevExpress.ui.widgets.editors/htmlEditorParts/toolbarModule.tests.js]: strictEqual?

File: js/ui/html_editor/modules/toolbar.js (hunk 1 above)
Comment (DokaRus) [reply js/ui/html_editor/modules/toolbar.js:306]: get "Insert Table" [dialog] form items

File: testing/tests/DevExpress.ui.widgets.editors/htmlEditorParts/api.tests.js (hunk 2 above)
Comment (DokaRus) [reply testing/tests/DevExpress.ui.widgets.editors/htmlEditorParts/api.tests.js:44]: Looks like tests don't includes an optional chaining polyfill.

**Review by ksercs — APPROVED** [review]

**Review by alexander-kotov-dx — APPROVED** [review]

**Pull request merged**